apps/intellij/src/main/kotlin/dev/nx/console/nxls/NxlsProcess.kt

```diff
@@ -95,8 +95,7 @@ class NxlsProcess(private val project: Project) {
             addParameter(lsp.path)
             addParameter("--stdio")
 
-            NodeCommandLineConfigurator.find(project.nodeInterpreter)
-                .configure(this, NodeCommandLineConfigurator.defaultOptions(project))
+            NodeCommandLineConfigurator.find(project.nodeInterpreter).configure(this)
```

Comment:
For clarity - this needed to be removed because if yarn pnp is enabled, then the nxls would try to be started with `yarn node`. Since the nxls is bundled already within the plugin, we should only run it with `node` even if its in a yarn pnp workspace.